packages/framework-integration-tests/integration/provider-unaware/end-to-end/cart.integration.ts

```diff
@@ -346,8 +345,8 @@ describe('Cart end-to-end tests', () => {
           `,
         })
 
-        console.log('Waiting 1 second for deletion to complete...')
-        await sleep(1000)
+        console.log('Waiting 4 second for deletion to complete...')
+        await sleep(4000)
```

Comment:
Nitpicking, but maybe..
```suggestion
        const secs = 4
        console.log('Waiting ${secs} second${secs > 1 ? 's' : ''} for deletion to complete...')
        await sleep(secs * 1000)
```